src/libs/actions/Welcome.ts

```diff
@@ -26,11 +37,13 @@ let currentUserAccountID;
  * - Whether we have loaded all reports the server knows about
  */
 function checkOnReady() {
-    if (!_.isBoolean(isFirstTimeNewExpensifyUser) || isLoadingReportData) {
+    if (typeof isFirstTimeNewExpensifyUser !== "boolean" || isLoadingReportData) {
```

Comment:
I know this is the direct translation of `!_.isBoolean()`, but wouldn't it make more sense if we just tested `isFirstTimeNewExpensifyUser === undefined`?